gsplat/cuda/csrc/backward.cu

```diff
@@ -79,15 +79,11 @@ __global__ void nd_rasterize_backward_kernel(
         const float sigma =
             0.5f * (conic.x * delta.x * delta.x + conic.z * delta.y * delta.y) +
             conic.y * delta.x * delta.y;
-        if (sigma < 0.f) {
-            valid = 0;
-        }
+        valid &= (sigma >= 0.f);
```

Comment:
out of curiosity is this for a reason?

Reply:
More thread coalescence, since they can all execute one & operation rather than branching with an if statement.